### ISSUE TITLE
Segmentation fault on JavaScript file

### ISSUE THREAD

**Issue by kevinsjoberg** [universal-ctags/ctags#3431]
The name of the parser:

The command line you used to run ctags:

```
$ ctags --options=NONE test.js
```

The content of input file:

```javascript
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});
Object.defineProperty(exports, 'getCallsite', {
  enumerable: true,
  get: function () {
    return _getCallsite.default;
  }
});

var _getCallsite = _interopRequireDefault(require('./getCallsite'));

function _interopRequireDefault(obj) {
  return obj && obj.__esModule ? obj : {default: obj};
}

```

The tags output you are not satisfied with:

```console
ctags: Notice: No options will be read from files or environment
ctags: Warning: ignoring null tag in test.js(line: 8)
Segmentation fault: 11
```

The version of ctags:

```console
$ ctags --version
Universal Ctags 5.9.0, Copyright (C) 2015-2022 Universal Ctags Team
Universal Ctags is derived from Exuberant Ctags.
Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
  Compiled: Jul  3 2022, 05:40:34
  URL: https://ctags.io/
  Optional compiled features: +wildcards, +regex, +gnulib_fnmatch, +gnulib_regex, +iconv, +option-directory, +xpath, +json, +interactive, +yaml, +case-insensitive-filenames, +packcc, +optscript
```

How do you get ctags binary:

Installed via the Homebrew tap (`universal-ctags/universal-ctags`).

**Comment by masatake** [comment]
The fix (#3428) was today.
Could you try the latest head of the master branch?

**Comment by kevinsjoberg** [comment]
@masatake looks like that did the trick. Latest `master` works fine. 👍
